README.md

```diff
@@ -119,6 +120,22 @@ The built-in configuration preset you get with `"extends": "tslint-react"` is se
       </button>
   );
   ```
+  - Rule options: _none_
+- `no-access-state-in-setstate`
+  - Forbids accessing component state with `this.state` within `this.setState`
+  calls, since React might batch multiple `this.setState` calls, thus resulting
+  in accessing old state. Enforces use of callback argument instead.
+    ```ts
+  // bad
```

Comment:
this did not get syntax highlighted / formatted correctly -- check out the markdown preview